docs/gettingstarted/quickstart.rst

```diff
@@ -0,0 +1,221 @@
+      -----------
+
+      +--------------------------------------------------------------------------+
+      | Runs on your laptop                                                      |
+      +--------------------------------------------------------------------------+
+      | Uses application and deployment configuration files                      |
+      +--------------------------------------------------------------------------+
+      | Deploys containers to a cluster of servers                               |
+      +--------------------------------------------------------------------------+
+
+      .. empty-div:: arrow-down center-block hidden-xs hidden-sm
+
+   .. parallel::
+
+      .. mobile-label::
+
+         Live
+
+      .. image:: images/nodes.png
+         :class: center-block offset-top img-responsive
+         :alt: Flocker Node diagram
+
+      Flocker Node
+      ------------
+
+      +--------------------------------------------------------------------------+
+      | Runs on each server in a cluster                                         |
+      +--------------------------------------------------------------------------+
+      | Links, ports and volumes work across hosts                               |
+      +--------------------------------------------------------------------------+
+      | After deployment, containers can move around                             |
+      +--------------------------------------------------------------------------+
+
+      .. empty-div:: arrow-down center-block
+
+   .. parallel::
+
+      .. mobile-label::
+
+         Local
+
+
+      Installation
+      ------------
+
+      .. tabs::
+
+         OS X
+         ^^^^
+
+         Install the flocker-cli client on your Mac (requires Homebrew):
+
+         .. code-block:: console
+
+            $ brew update && \
+              brew tap clusterhq/flocker && \
+              brew install flocker-0.3.2
+
+         Linux
+         ^^^^^
+
+         Install the flocker-cli client on your Linux machine:
+
+         .. code-block:: console
+
+            $ virtualenv flocker-tutorial && \
+              flocker-tutorial/bin/pip install --upgrade pip && \
+              flocker-tutorial/bin/pip install --quiet flocker-cli
+
+      .. empty-div:: arrow-down center-block invisible
+
+   .. parallel::
+
+      .. mobile-label::
+
+         Live
+
+
+      Installation
+      ------------
+
+      .. tabs::
+
+         Vagrant
+         ^^^^^^^
+
+         Simulate a Flocker cluster with virtual machines on your laptop (requires Vagrant, VirtualBox):
+
+         .. code-block:: console
+
+            $ git clone \
+              https://github.com/clusterhq/vagrant-flocker && \
+              cd vagrant-flocker && \
+              vagrant up
+
+         AWS
+         ^^^
+
+         Please see our separate `AWS install instructions <http://docs.clusterhq.com/en/latest/gettingstarted/installation.html#using-amazon-web-services>`_ to get started.
```

Comment:
Uh.  This ... seems like it sort of defeats the purpose of the complex, specific layout we've gone to very much trouble to create.